hack/cozyreport.sh

```
@@ -64,6 +64,27 @@ if kubectl get crd certificates.cert-manager.io >/dev/null 2>&1; then
   fi
 fi
 
+# -- trust-manager module
+if kubectl get crd bundles.trust.cert-manager.io >/dev/null 2>&1; then
+  echo "Collecting trust-manager state..."
+  DIR=$REPORT_DIR/trust-manager
+  mkdir -p $DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)